nixos/modules/services/backup/restic.nix

```diff
@@ -465,7 +465,7 @@ in
         # set same environment variables as the systemd service
         ${lib.pipe config.systemd.services."restic-backups-${name}".environment [
           (lib.filterAttrs (n: v: v != null && n != "PATH"))
-          (lib.mapAttrsToList (n: v: "${n}=${v}"))
+          (lib.mapAttrsToList (n: v: lib.toShellVar n "${v}"))
           (lib.concatStringsSep "\n")
```

Comment:
Can't we save those two lines by using lib.toShellVars?

Reply:
See <https://github.com/NixOS/nixpkgs/pull/395005#discussion_r2023391417> above.